### PR TITLE
CloudUploader: require drift signature parameter to ensure clock drift can be addressed

### PR DESCRIPTION
**Pull request by mikesherov**

### CODE REVIEW

**Comment by mrjoelkemp** [comment]
:+1:

File: Component/CloudUploader.js [hunk 1]
```diff
@@ -226,7 +226,7 @@ define([
         throw new Error('Please provide a proper `request` configuration property');
       }
 
-      if (!options.signature || !options.signature.endpoint) {
+      if (!options.signature || !options.signature.endpoint || !options.signature.params  || !('drift' in options.signature.params)) {
```
Comment (Attamusc) [inline Component/CloudUploader.js]: So a drift **has** to be provided now?

Comment (mikesherov) [inline Component/CloudUploader.js]: yes

Comment (mikesherov) [inline Component/CloudUploader.js]: hmmm, maybe this should be a top level key that I transform on the innards.

**Comment by Attamusc** [comment]
:+1:

**Pull request merged**